src/spikeinterface/core/base.py

```diff
@@ -977,16 +1005,16 @@ def save_to_zarr(
         return cached
 
 
-def _make_paths_relative(d, relative) -> dict:
+def _make_paths_relative(d, relative, copy=True) -> dict:
     relative = str(Path(relative).resolve().absolute())
     func = lambda p: os.path.relpath(str(p), start=relative)
-    return recursive_path_modifier(d, func, target="path", copy=True)
+    return recursive_path_modifier(d, func, target="path", copy=copy)
 
 
-def _make_paths_absolute(d, base):
+def _make_paths_absolute(d, base, copy=True):
```

Comment:
This is a bad idea to expose at top level.
The first call of the recursive must be a copy always to avoid complicated inplace modification.
I do not see where we need copy=False.

Here th logic was copy the first call always and then internally no copy.

Reply:
I was experimenting with this. Not important to me. Will revert. Sorry for the distraction.

Reply:
ok

Reply:
Done.